lib/commands.js

```diff
@@ -61,8 +57,12 @@ commands.list.forEach(function (command) {
             }
             return this.internal_send_command(new Command(command, arr, callback));
         };
+        //alias commands with illegal function names (e.g. NR.RUN becomes NR_RUN and nr_run)
+        if (commandName !== command) {
+            RedisClient.prototype[commandName.toUpperCase()] = RedisClient.prototype[commandName] = RedisClient.prototype[command];
+        }
```

Comment:
1) It is not a illegal function name as it is also property and those may contain anything.
2) I would indeed prefer not to add the alternative writing but I now understand your intention behind it. If this should be added, it must also be documented. That should be the case for `addCommand` in general.